packages/shipjs/bin/shipjs

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env node
 
-require = require('esm')(module);
-(async function() {
+(async function () {
```

Comment:
not sure about the history but this likely was an async iife to ensure all errors are caught. Not sure that's needed really tbh. Can be changed in a separate PR and release to be just a regular sync module maybe?

Reply:
I'm easy with however you want to do it. I was just trying to keep changes to a minimum!